tutornotes/templates/notes/build/notes/Dockerfile

```diff
@@ -1,20 +1,21 @@
 # syntax=docker/dockerfile:1.4
-FROM docker.io/ubuntu:20.04
+FROM docker.io/python:3.12-slim-bookworm
 
 ENV DEBIAN_FRONTEND=noninteractive
 RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
   --mount=type=cache,target=/var/lib/apt,sharing=locked \
   apt update && \
   apt upgrade -y && \
   apt install -y \
-    language-pack-en \
+    locales \
     git \
-    python3 \
-    python3-pip \
-    python3-venv \
-    libmysqlclient-dev \
-    pkg-config
-RUN ln -s /usr/bin/python3 /usr/bin/python
+    python3-dev \
+    build-essential \
+    default-libmysqlclient-dev \
+    pkg-config && \
+  sed -i '/en_US.UTF-8/s/^# //g' /etc/locale.gen && \
```

Comment:
why is this needed?

Reply:
Any links/references?

Reply:
@DawoudSheraz please check the link [here](https://stackoverflow.com/a/28406007).